pss/pubsub.go

```diff
@@ -65,6 +68,8 @@ func (p *PubSub) Register(topic string, prox bool, handler func(msg []byte, p *p
 
 // Send sends a message using pss SendRaw
 func (p *PubSub) Send(to []byte, topic string, msg []byte) error {
+	metrics.GetOrRegisterCounter("pss.pubsub.send.num", nil).Inc(1)
```

Comment:
No need to include a `counter` where you have a `timer` - the `timer` data structure already includes a counter, that would be `pss.pubsub.send.count` in this example.

Reply:
Oh! that's nice. I was wondering why the framework failed when trying to register a counter with the same name. I will change it.

Reply:
Yes, when you use the same name, then they collide, and it panics if I remember correctly.

Reply:
Well, not exactly, the value is `pss.pubsub.send.span` and the `select` field(count).